src/developer/web-api/route.md

```diff
@@ -140,6 +130,22 @@ In the example shown below, we are configuring a route with `http-basic` authent
 
 Custom authorities allows a DHIS2 client that does not have the rights to manage the route to still be able to run it. This enables the route to be run from your app.
 
+### Running a route with custom response timeout
+
+The pre-configured response timeout for the origin when running a route is 10 seconds. A response taking longer than this will result in the DHIS2 server returning a gateway timeout error to the client. However, this timeout can be adjusted when creating or updating a route as demonstrated below:
+
+```json
+{
+  "name": "Postman Echo",
+  "code": "postman-post",
+  "disabled": false,
+  "url": "https://postman-echo.com/post",
+  "responseTimeout": 5
```

Comment:
I guess it is a bit late now but the naming here isn't the best. For a timeout time given without any unit in the name I would always assume it is milliseconds. So any time that isn't milliseconds should have the unit in the name, e.g. `responseTimeoutSeconds`. Otherwise people have to find and read the documentation just to understand what number to put.